src/handlers/ecommerce/activation.ts

```diff
@@ -98,7 +99,16 @@ const return404ErrorDominioSconosciuto = (res: any): void => {
   res.status(404).send(error404DominioSconosciuto());
 };
 
+const return404ResourceNotFound = (res: any): void => {
+  logger.info("[Activation ecommerce] - Return 404 Resource not found");
+  res.status(404).send(error404ResourceNotFound());
```

Comment:
Wouldn’t a 401 be better to simulate the policy error ?      
https://github.com/pagopa/pagopa-infra/blob/cf80cc36e5c1475a8de378a8e6e347584f8f05f1/src/domains/ecommerce-app/api/ecommerce-checkout/v1/_transaction_policy.xml.tpl#L25

Reply:
It isn't properly an unauthorized. 
I tried myself without passing this parameter and it result in a 404 resource not found.
The request is build by definition with that query parameters. If you didn't pass it, you are not handling the right resource. Please remember also that the scope of this check is to be sure that the parameter exists